### PR TITLE
add port threshold

### PR DESCRIPTION
**Pull request by parrasajad**
example:
```console
$cat hostlist.txt | naabu -pts 5
```
will skip hosts with 5 or more than 5 ports open

### CODE REVIEW

File: v2/pkg/runner/options.go [hunk 1]
```diff
@@ -102,6 +103,7 @@ func ParseOptions() *Options {
 		flagSet.StringVarP(&options.TopPorts, "tp", "top-ports", "", "top ports to scan (default 100)"),
 		flagSet.StringVarP(&options.ExcludePorts, "ep", "exclude-ports", "", "ports to exclude from scan (comma-separated)"),
 		flagSet.StringVarP(&options.PortsFile, "pf", "ports-file", "", "list of ports to scan (file)"),
+		flagSet.IntVarP(&options.PortThreshold, "pts", "port-threshold", 0, "port threshold to skip port scan for the host"),
```
Comment (Mzack9999) [inline v2/pkg/runner/options.go:106]: I think we can add validation in `Options.validateOptions: the threshold should be between 0 and 65535.

**Review by Mzack9999 — CHANGES_REQUESTED** [review]

**Review by Mzack9999 — APPROVED** [review]
```console
$ go run . -pts 5 -host scanme.sh
...
[INF] Found 4 ports on host scanme.sh (128.199.158.128)
scanme.sh:80
scanme.sh:444
scanme.sh:22
scanme.sh:443
$ go run . -pts 3 -host scanme.sh -verbose
...
root privileges
[DBG] Skipping 128.199.158.128, Threshold reached
```

**Review by ehsandeep — CHANGES_REQUESTED** [review]
On Mac, found results are not displayed:

```console
go run . -pts 5 -host scanme.sh -v

                  __
  ___  ___  ___ _/ /  __ __
 / _ \/ _ \/ _ \/ _ \/ // /
/_//_/\_,_/\_,_/_.__/\_,_/ v2.1.0

		projectdiscovery.io

Use with caution. You are responsible for your actions
Developers assume no liability and are not responsible for any misuse or damage.
[DBG] Using host 128.199.158.128 for enumeration
[INF] Running CONNECT scan with non root privileges
[DBG] Skipping 128.199.158.128, Threshold reached 
```

**Comment by ehsandeep** [comment]
also with SYN scan, `-pts` is not working:

```console
sudo go run . -pts 2 -host scanme.sh -silent

scanme.sh:444
scanme.sh:22
scanme.sh:25
scanme.sh:80
scanme.sh:443
scanme.sh:445
```

**Comment by Mzack9999** [comment]
- [ ] Please update `REAME.MD`

**Review by ehsandeep — APPROVED** [review]

**Review by Mzack9999 — APPROVED** [review]

**Pull request merged**